prow/config.yaml

```diff
@@ -18,6 +18,7 @@ plank:
       gcs_configuration:
         bucket: "istio-prow"
         path_strategy: "explicit"
+      blobless_fetch: true
```

Comment:
can you details about this?

Reply:
https://github.com/kubernetes/test-infra/issues/26590

tl;dr `git clone` faster, should have zero downsides I think (or if I am wrong we can just revert)

Reply:
TIL, thanks!